### PR TITLE
Fixed python string formating to avoid error

### PR DESCRIPTION
**Pull request by jburns11**
Hello All,

I was working on a python program to automate testing, and I was attempting to use pprzlink to connect to the vehicle during this process. When I included the required files (i.e., from pprzlink.ivy import IvyMessagesInterface) i got an syntax error on line that I replaced. I simply changed the statement ...
f'{pid}_{sequence_number}' 
... to the line ...
'%_%'.format(pid, sequence_number)

I replaced this line in V1.0 and V2.0

Here is the error message....

Traceback (most recent call last):
  File "automate.py", line 14, in <module>
    from pprzlink.ivy import IvyMessagesInterface
  File "/home/jb/paparazzi/var/lib/python/pprzlink/ivy.py", line 12, in <module>
    from pprzlink.request_uid import RequestUIDFactory
  File "/home/jb/paparazzi/var/lib/python/pprzlink/request_uid.py", line 12
    yield f'{pid}_{sequence_number}'
                                   ^
SyntaxError: invalid syntax

### CODE REVIEW

**Comment by Fabien-B** [comment]
Hello,
The syntax used is indeed new in python 3.6, it's good to replace it with a more available syntax.
But are you sure your syntax is good ? If I test it:
```
>>> '%d_%d'.format(10,42)
'%d_%d'
```
You can replace it with this syntax instead:
```
>>> '{}_{}'.format(10,42)
'10_42'
```

File: lib/v1.0/python/pprzlink/request_uid.py [hunk 1]
```diff
@@ -26,7 +26,7 @@ def _unique_id_generator(cls):
         sequence_number = 0
         while True:
             sequence_number = sequence_number + 1
-            yield f'{pid}_{sequence_number}'
+            yield '%d_%d'.format(pid, sequence_number)
```
Comment (Fabien-B) [inline lib/v1.0/python/pprzlink/request_uid.py:29]: '{}_{}'.format(pid, sequence_number)

File: lib/v2.0/python/pprzlink/request_uid.py [hunk 2]
```diff
@@ -26,7 +26,7 @@ def _unique_id_generator(cls):
         sequence_number = 0
         while True:
             sequence_number = sequence_number + 1
-            yield f'{pid}_{sequence_number}'
+            yield '%d_%d'.format(pid, sequence_number)
```
Comment (Fabien-B) [inline lib/v2.0/python/pprzlink/request_uid.py:29]: '{}_{}'.format(pid, sequence_number)

**Review by Fabien-B — CHANGES_REQUESTED** [review]

**Comment by jburns11** [comment]
Thank you for the suggestions!! I made the fix and re-committed!

**Review by Fabien-B — APPROVED** [review]

**Pull request merged**